int32    task_ID
Point2d  task_pose
